tests/test_packages/test_skills_integration/test_simple_oracle.py

```diff
@@ -156,6 +156,10 @@ def test_oracle(
             f"{DEFAULT_FETCH_LEDGER_ADDR}:{DEFAULT_FETCH_LEDGER_REST_PORT}",
         )
 
+        # use alternate port for prometheus connection
+        setting_path = "vendor.fetchai.connections.prometheus.config.port"
+        self.set_config(setting_path, 9091, type_="int")
```

Comment:
What was the port used before for prometheus connection?

Reply:
It was `9090` which is exposed in the new 'fetchd` container, so this was just to remove that conflict.